administrator/components/com_workflow/Model/TransitionsModel.php

```diff
@@ -119,40 +134,41 @@ public function getListQuery()
 
 		$select = $db->quoteName(
 			array(
-			'transition.id',
-			'transition.title',
-			'transition.published',
+			't.id',
+			't.title',
+			't.published',
+			't.ordering',
 		)
 		);
 		$select[] = $db->qn('f_state.title', 'from_state');
 		$select[] = $db->qn('t_state.title', 'to_state');
 		$joinTo = $db->qn('#__workflow_states', 't_state') .
-			' ON ' . $db->qn('t_state.id') . ' = ' . $db->qn('transition.to_state_id');
+			' ON ' . $db->qn('t_state.id') . ' = ' . $db->qn('t.to_state_id');
 
 		$query
 			->select($select)
-			->from($db->qn('#__workflow_transitions', 'transition'))
+			->from($db->qn('#__workflow_transitions', 't'))
 			->leftJoin(
-				$db->qn('#__workflow_states', 'f_state') . ' ON ' . $db->qn('f_state.id') . ' = ' . $db->qn('transition.from_state_id')
+				$db->qn('#__workflow_states', 'f_state') . ' ON ' . $db->qn('f_state.id') . ' = ' . $db->qn('t.from_state_id')
 			)
 			->leftJoin($joinTo);
 
 		// Filter by extension
 		if ($workflowID = (int) $this->getState('filter.workflow_id'))
 		{
-			$query->where($db->qn('transition.workflow_id') . ' = ' . $workflowID);
+			$query->where($db->qn('t.workflow_id') . ' = ' . $workflowID);
 		}
 
 		$status = $this->getState('filter.published');
 
 		// Filter by condition
 		if (is_numeric($status))
 		{
-			$query->where($db->qn('transition.published') . ' = ' . (int) $status);
+			$query->where($db->qn('t.published') . ' = ' . (int) $status);
 		}
 		elseif ($status == '')
 		{
-			$query->where($db->qn('transition.published') . " IN ('0', '1')");
+			$query->where($db->qn('t.published') . " IN ('0', '1')");
```

Comment:
Why using strings? just use ```(0, 1)```

Reply:
because what i remember with numbers there was an error, i do not know why exactly because on that column is tinyint(1) ...